src/Symfony/Bridge/Doctrine/ManagerRegistry.php

```diff
@@ -51,6 +52,13 @@ protected function resetService($name)
         }
         $manager = $this->container->get($name);
 
+        if ($manager instanceof LazyObjectInterface) {
```

Comment:
This interface doesn't exist in the 5.4 branch so not sure on the best way to write a test for it so open to ideas 😄